src/pal/pal_apple.h

```diff
@@ -23,6 +23,8 @@ namespace snmalloc
      */
     static constexpr uint64_t pal_features = PALBSD::pal_features;
 
+    static constexpr size_t OS_PAGE_SIZE = 0x1000;
```

Comment:
`PalApple` inherits from `PALBSD`.  Why do you need this here, but not in other *BSD PALs.

Reply:
Mm, I probably don't and added this one first to appease CI.  Let's see what happens without.